tools/train_classifier.py

```diff
@@ -124,26 +124,30 @@ def train_model(path_in, path_out, model_name, model_version, num_layers_to_fine
     # Find label names
     label_names = os.listdir(directories.get_videos_dir(path_in, 'train'))
     label_names = [x for x in label_names if not x.startswith('.')]
-    label_counting = ['counting_background']
 
-    for label in label_names:
-        label_counting += [f'{label}_position_1', f'{label}_position_2']
+    with open(os.path.join(path_in, 'project_config.json')) as f:
+        project_config = json.load(f)
```

Comment:
This function can still be called as a stand-alone script, so the `project_config.json` doesn't necessarily exist.
Also we have a `PROJECT_CONFIG_FILE` constant that could be used here.